src/hotspot/cpu/aarch64/assembler_aarch64.hpp

```diff
@@ -2706,26 +2717,28 @@ template<typename R, typename... Rx>
 
 #undef INSN
 
+  void fp_arithmetic(Instruction_aarch64 &current_insn, FloatRegister Vd,
+                     SIMD_Arrangement T, FloatRegister Vn, FloatRegister Vm,
+                     int op1, int op2, int op3);
+
 // Advanced SIMD three same
```

Comment:
```suggestion
    // Advanced SIMD three same

    void adv_simd_three_same(Instruction_aarch64 &current_insn, FloatRegister Vd,
                     SIMD_Arrangement T, FloatRegister Vn, FloatRegister Vm,
                     int op1, int op2, int op3);

```
Always match names with names in the Architecture Reference Manual where we can.